spec/cohortManager.spec.js

```diff
@@ -0,0 +1,217 @@
+const CohortManager = require('../src/cohortManager.js')
+const Cohort = require('../src/cohort.js')
+const Student = require('../src/student.js')
+
+describe('The Cohort Manager should be able to', () => {
+  let cm
+
+  beforeEach(() => {
+    cm = new CohortManager()
+  })
+
+  it('create a new cohort with a name', () => {
+    // set up
+    const expected = [new Cohort('test')]
+    // execute
+    cm.createCohort('test')
+    // verify
+    expect(cm.cohorts).toEqual(expected)
+  })
+
+  it('create multiple cohorts', () => {
+    // set up
+    const expected = [
+      new Cohort('test1'),
+      new Cohort('test2'),
+      new Cohort('test3')
+    ]
+    // execute
+    cm.createCohort('test1')
+    cm.createCohort('test2')
+    cm.createCohort('test3')
+    // verify
+    expect(cm.cohorts).toEqual(expected)
+  })
+
+  it('should display an error message if a cohort was already created', () => {
+    // set up
+    cm.createCohort('test1')
+    const expected = 'Cohort already exists'
```

Comment:
you could look into throwing up an error and then catching it in the test runner; that would make the code much cleaner and extendable:

```js
if(errorCondition) {
   throw new Error('Error message....')
}
```